breathe/renderer/sphinxrenderer.py

```diff
@@ -1647,6 +1733,12 @@ def dispatch_memberdef(self, node) -> List[Node]:
             return self.visit_typedef(node)
         if node.kind == "variable":
             return self.visit_variable(node)
+        if node.kind == "property":
+            # Note: visit like variable for now
+            return self.visit_variable(node)
+        if node.kind == "event":
+            # Note: visit like variable for now
+            return self.visit_variable(node)
```

Comment:
Is there a special way we should handle properties and events? It seems to work like this.

Reply:
It depends, does it render how it is supposed to? E.g., similar to how it looks if one would write the raw rst manually?

Reply:
For me it renders as it should